python/ray/actor.py

```diff
@@ -142,7 +142,7 @@ def _remote(self, args=None, kwargs=None, name="", num_returns=None):
         def invocation(args, kwargs):
             actor = self._actor_hard_ref or self._actor_ref()
             if actor is None:
-                raise RuntimeError("Lost reference to actor")
+                raise RuntimeError("Lost reference to actor. If you are trying to reference an actor via ray.get_actor('name'). Then set it as a variable first.")
```

Comment:
It's not just named actors. It's also regular actors. See https://github.com/ray-project/ray/issues/6265

```suggestion
                raise RuntimeError("Lost reference to actor. If you are trying to use an actor handle immediately after creating or fetching it, you may need to store the actor handle in a variable first.")
```

Reply:
Even better would be to just make it work.